.travis.yml

```diff
@@ -15,19 +15,23 @@ before_install:
   - 'if [ -n "$encrypted_f942601034d6_key" -a -n "$encrypted_f942601034d6_iv" ]; then openssl aes-256-cbc -K $encrypted_f942601034d6_key -iv $encrypted_f942601034d6_iv -in tests/secrets.tar.enc -out tests/secrets.tar -d; cd tests && tar xvf secrets.tar ; fi'
 sudo: false
 
-script: 'SPEC_OPTS="--format documentation" bundle exec rake validate lint spec strings:generate reference'
+script: 'bundle exec rake $CHECK'
 
 matrix:
   fast_finish: true
   include:
   - rvm: 2.4.4
```

Comment:
Puppet 5 is now using ruby 2.4.5. Want to go ahead and make that part of this?

https://puppet.com/docs/puppet/5.5/about_agent.html

https://puppet.com/docs/puppet/latest/about_agent.html

Reply:
I replaced all 2.4.4 with 2.4.5.